addons/website/views/snippets/s_sidegrid.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_sidegrid" name="Side Grid">
+    <section class="s_sidegrid pt56 pb56">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="13">
+                <div class="o_grid_item g-col-lg-5 g-height-9 col-lg-3" style="z-index: 1; grid-area: 1 / 8 / 10 / 13; --grid-item-padding-x: 24px">
+                    <h1 class="display-4">Experience<br/>the real<br/>innovation</h1>
+                    <p class="lead"><br/>Every groundbreaking innovation, whether meticulously engineered or born from spontaneous creativity, contains stories waiting to be discovered.<br/><br/></p>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-height-4 g-col-lg-4 col-lg-2" style="z-index: 5; grid-area: 1 / 1 / 5 / 5;">
+                    <img class="img img-fluid rounded" src="/web/image/website.s_sidegrid_default_image_1" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image o_snippet_mobile_invisible g-height-9 g-col-lg-3 col-lg-2 d-lg-block d-none" style="z-index: 2; grid-area: 1 / 5 / 10 / 8;">
+                    <img class="img img-fluid rounded" src="/web/image/website.s_sidegrid_default_image_2" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image o_snippet_mobile_invisible g-height-9 g-col-lg-4 col-lg-2 d-lg-block d-none" style="z-index: 4; grid-area: 5 / 1 / 14 / 5;">
+                    <img class="img img-fluid rounded" src="/web/image/website.s_sidegrid_default_image_4" alt=""/>
+                </div>
+                <div class="testanso o_grid_item o_grid_item_image o_snippet_mobile_invisible g-height-4 g-col-lg-8 col-lg-3 d-lg-block d-none" style="z-index: 3; grid-area: 10 / 5 / 14 / 13;">
```

Comment:
@anso-odoo probably a leftover? "testanso" :)

Reply:
Indeed, i'll remove that 👍